lib/oauth2/model/authorization.rb

```diff
@@ -104,6 +106,14 @@ def self.for(owner, client, attributes = {})
         end
       end
 
+      def self.find_by_jwt(jwt)
+        begin
```

Comment:
This could be `User.find_by_id(jwt[:sub])` which will return either the user or nil if it doesn't exist.